src/utils/useMapEvents.ts

```diff
@@ -0,0 +1,12 @@
+import { LeafletEventHandlerFnMap, Map } from 'leaflet'
+import useMapInstance from './useMapInstance'
+import useEvents from './useEvents'
+
+/**
+ * Uses useEvents hook with mapInstance
+ * @param events Leaflet event handlers, check https://leafletjs.com/reference-1.6.0.html#map-event
+ */
+export default (events?: LeafletEventHandlerFnMap) => {
```

Comment:
Although it is very compact to export an anonymous function by default, I am afraid that if en exception occurs this will show up as `<anonymous function>` instead of `useMapEvents` in stack traces. That might be problematic for users of the library when they need to understand where an error is coming from.

Reply:
ah fair enough! will name it